servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -23,19 +23,24 @@ nessie.server.should-sendstack-trace-to-api-client=true
 nessie.version.store.type=INMEMORY
 
 ## JGit version store specific configuration
-### Which type of jgit version store to use: INMEMORY or DISK. in memory is best for tests and DISK is recommended. Dynamo is an experimental backend
+### Which type of jgit version store to use: INMEMORY or DISK. in memory is best for tests and DISK is recommended. Dynamo and Mongo are experimental backends
 nessie.version.store.jgit.type=DISK
 ### where to put your git repository. Only used if nessie.version.store.jgit.type=JGIT
 nessie.version.store.jgit.directory=/tmp/jgit
 
-## Dyanmo version store specific configuration
+## Dynamo version store specific configuration
 ### should Nessie create its own dynamo tables
 nessie.version.store.dynamo.initialize=false
 ### table names for ref, tree and value tables respectively
 nessie.version.store.dynamo.refTableName=nessie_refs
 nessie.version.store.dynamo.treeTableName=nessie_objects
 nessie.version.store.dynamo.valueTableName=nessie_objects
 
+## MongoDB version store specific configuration
+### database name for Nessie object storage
+nessie.version.store.mongodb.databaseName=nessie
```

Comment:
Let's not support using Mongo in configuration until it can actually be used. Please pull out changes related to using the Mongo connector since it doesn't actually work right now.

Reply:
Done.